app/wama.web/Views/AdminConsole/UserTool/Index.cshtml

```diff
@@ -3,8 +3,37 @@
 
 <h2>User account summaries</h2>
 
-<h3>Accounts pending approval</h3>
-@Html.Partial("UserAccountTableDisplay")
 
-<h3>Suspended Accounts</h3>
-@Html.Partial("UserAccountTableDisplay")
+
+<center>
+    <h3>Accounts pending approval</h3>
+</center>
+<div>
+    <h4>Pending Administrators</h4>
```

Comment:
Instead of dumping all pending user on one table,I am trying to categorize them to know which user is pending approval.Is it Patron/Manager/Employee/Administrator?